instrumentation/opentelemetry-sdk-extension-autoconfigure-1.28.0/build.gradle

```diff
@@ -0,0 +1,29 @@
+plugins {
+    id("java")
+}
+
+dependencies {
+    implementation(project(":agent-bridge"))
+    implementation(project(":newrelic-api"))
+    implementation(project(":newrelic-weaver-api"))
+    implementation("io.opentelemetry:opentelemetry-sdk-extension-autoconfigure:1.28.0")
```

Comment:
What's the thinking behind this specific version? Are we trying to use the earliest version that has the method we need to instrument?

Reply:
I just picked a version.  I could go back further, but I also think it's reasonable to say that this works with newer versions of the SDK.